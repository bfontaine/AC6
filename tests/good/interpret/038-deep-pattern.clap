val fst_some = case { Some [{ _1 -> x, _2 -> _ }] => x }
val z = fst_some (Some [{ _1 <- 42, _2 <- 0 }])

