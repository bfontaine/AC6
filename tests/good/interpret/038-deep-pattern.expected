:- fst_some = <code>
:- z = 42
